### DISCUSSION TITLE
Access BirdNET-Pi within Home LAN

### DISCUSSION THREAD

**Discussion by Rochambeaux** [mcguirepr89/BirdNET-Pi#434] [General]
What's needed to access the browser UI from another system within my home LAN?  I have the Pi setup as a headless device and can access via RDP but I'd to be able to listen to the recordings on my primary system.

**Comment by CaiusX** [comment]
Try http://birdnetpi.local/ from another machine/device on the same LAN.

**Reply by Rochambeaux** [reply]
Doh!  missed the last "/".  Thanks!
